house_of_refuge/frontend/src/components/ResourceRow.js

```diff
@@ -149,6 +160,31 @@ export const ResourceRow = ({resource, isExpanded, onMatch, user, activeSub, com
     setAvailableFrom(newDate);
   };
 
+  const isTaken = () => {
```

Comment:
How about `isCallingOrBooked`? "taken" is introducing yet another term which might be not so easy to intuitively distinguish from "booked" (in common speech booked implies taken AFAIU)

Reply:
OK, I can see below "taken" is already an existing term. Can you confirm that its meaning here is consistent with how it was before?